unittests/runtime/Array.cpp

```diff
@@ -47,7 +47,7 @@ extern "C" void swift_arrayInitWithCopy(OpaqueValue *dest,
   initialize_pod_witness_table_size_uint32_t_stride_uint64_t(pod_witnesses);   \
   uint64_t srcArray[3] = {0, 1, 2};                                            \
   uint64_t destArray[3] = {0x5A5A5A5AU, 0x5A5A5A5AU, 0x5A5A5A5AU};             \
-  FullOpaqueMetadata testMetadata = {{&pod_witnesses},                         \
+  FullOpaqueMetadata testMetadata = {{&pod_witnesses},            \
```

Comment:
I think the `{ nullptr },` went missing here.

Reply:
That is on purpose. `OpaqueMetadata` does not have a layout string.

Reply:
Oh, got it! Then there’s an unnecessary white space change here.